Resources/Prototypes/_Impstation/Heretic/Entities/Objects/Specific/heretic.yml

```diff
@@ -0,0 +1,48 @@
+- type: entity
+  parent: BaseItem
+  id: EldritchTome
+  name: Eldritch Tome
```

Comment:
i dont typically review yaml but i was looking to see where `EldritchItem` was used - item names should be in lowercase